### PR TITLE
tools/ctl: add caller ID for `pd-ctl`

### PR DESCRIPTION
**Pull request by HuSharp**
### What problem does this PR solve?

Issue Number: Ref #7300

### What is changed and how does it work?

After replacing HTTP with SDK #7490, we need to add the `calledID` explicitly to keep consistent with #4491

And source ID can be consistent with caller ID, #7565 

```commit-message
```

### Check List

Tests

- Unit test

### Release note

```release-note
None.
```

### CODE REVIEW

**Comment by HuSharp** [comment]
@okJiang PTAL, thx!

File: tools/pd-ctl/tests/global_test.go [hunk 1]
```diff
@@ -16,33 +16,46 @@ package tests
...
 	"go.uber.org/zap"
 )
 
-const pdControlCallerID = "pd-ctl"
-
 func TestSendAndGetComponent(t *testing.T) {
 	re := require.New(t)
 	handler := func(context.Context, *server.Server) (http.Handler, apiutil.APIServiceGroup, error) {
 		mux := http.NewServeMux()
+		mux.HandleFunc("/pd/api/v1/cluster", func(w http.ResponseWriter, r *http.Request) {
```
Comment (HuSharp) [inline tools/pd-ctl/tests/global_test.go:40]: For now, only cluster command is replaced by PD HTTP SDK

File: tools/pd-ctl/tests/global_test.go [hunk 2]
```diff
@@ -16,33 +16,46 @@ package tests
...
+			callerID := apiutil.GetCallerIDOnHTTP(r)
+			for k := range r.Header {
+				log.Info("header", zap.String("key", k))
+			}
+			log.Info("caller id", zap.String("caller-id", callerID))
```
Comment (JmPotato) [inline tools/pd-ctl/tests/global_test.go:45]:
```suggestion
```

**Review by okJiang — APPROVED** [review]
Please add a comment to explain why we need two handle for testing. Rest lgtm

**Pull request merged**